firmware/src/main.cpp

```diff
@@ -1,17 +1,22 @@
+#include <pico/stdlib.h> // for uart printing
 #include <cstring>
-#include <harp_c_app.h>
-#include <harp_synchronizer.h>
-#include <core_registers.h>
-#include <reg_types.h>
-#include <bme688.h>
+#include <stdio.h>
 #include <config.h>
 #ifdef DEBUG
```

Comment:
Having `#ifdef DEBUG` statements is OK, but let's keep the resulting behavior changes minimal rather than compiling with `-DDEBUG` resulting in very different behavior. Now that we have transactions working with the sensor, let's consider removing the debug sections that completely change the device behavior, or let's delegate that to another file for code readability.